application/libraries/admin/quexmlpdf.php

```diff
@@ -1966,7 +1966,11 @@ public function createqueXML($quexml)
                     }
                 }
 
-                $qtmp['title'] = $sl.$qcount.$this->questionTitleSuffix;
+                if (Yii::app()->getConfig('quexmlusequestiontitleasid') == true) {
+                    $qtmp['title'] = ((string) $qu->response->attributes()->varName).$this->questionTitleSuffix;
+                } else {
+                    $qtmp['title'] = $sl.$qcount.$this->questionTitleSuffix;
```

Comment:
Didn't PSR-12 require space between dots?

Reply:
I don't get any warning on the editor about that. We just follow previous code style here.

Reply:
It's part of PSR-12:

> All binary arithmetic, comparison, assignment, bitwise, logical, string, and type operators MUST be preceded and followed by at least one space:

https://www.php-fig.org/psr/psr-12/